pcsc-sys/src/lib.rs

```diff
@@ -41,7 +41,13 @@ pub type LONG = i32;
 #[cfg(target_os = "macos")]
 pub type ULONG = u32;
 
+#[cfg(target_os = "windows")]
+pub type SCARDCONTEXT = *const c_ulong;
```

Comment:
Looking [here](https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx), it doesn't look like `LONG_PTR` should be an actual pointer. I think it'd be best to follow [winapi's lead](https://github.com/retep998/winapi-rs/blob/ad224c13346b12ecc565d6333e5c2cf890360e20/src/shared/basetsd.rs#L36) and define it as `isize`.

Defining it to be an integral type is preferable regardless - the type is not opaque, so it'd be difficult to work with the library in a portable way if it's integral on one platform and a pointer on another.

Reply:
Agreed